app/components/elements/modal/reducer.js

```diff
@@ -6,7 +6,7 @@ const modalReducer = (state, {type, payload}) => {
       return {...state, modal: payload.name}
 
     case CLOSE_MODAL:
-      return {...state, modal: undefined}
+      return {...state, modal: null}
```

Comment:
what's the practical implication of this? null is an object still, where as undefined is a unique type. I've always felt the latter was a more explicit unsetting of a variable or property.

Reply:
oh yeah should have left a comment. for some reason the watchStore in `modal/index.js` was not getting fired when modal was set to undefined so the class name that stops scrolling in the body wasn't getting removed. Replacing undefined with null solved it.